pages/admin.tsx

```diff
@@ -24,6 +25,27 @@ const Admin = ({ profile }) => {
               or find any errors, hit up the engineers in{" "}
               <b>#{new Date().getFullYear()}-engineering</b>
             </p>
+
+            <PaddedFlex direction="column">
+              <PaddedFlex direction="row">
+                <PaddedSubTitle>hacksc.com: </PaddedSubTitle>
+                <ReactMarkdown>
+                  [![HackSC.com](https://betteruptime.com/status-badges/v1/monitor/50yh.svg)](https://betteruptime.com/?utm_source=status_badge)
```

Comment:
Is there a reason we need `react-markdown`? Can you just `<img src='https://betteruptime.com/status-badges/v1/monitor/50yh.svg' alt=''">`?